security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -240,10 +240,8 @@ func (p *Plugin) constructFederatedTokenRequest(parameters security.StsRequestPa
 		dReq, _ := http.NewRequest("POST", federatedTokenEndpoint, bytes.NewBuffer(dJSONQuery))
 		dReq.Header.Set("Content-Type", contentType)
 
-		if pluginLog.DebugEnabled() {
-			reqDump, _ := httputil.DumpRequest(dReq, true)
-			pluginLog.Debugf("Prepared federated token request: \n%s", string(reqDump))
-		}
+		reqDump, _ := httputil.DumpRequest(dReq, true)
+		pluginLog.Debugf("Prepared federated token request: \n%s", string(reqDump))
```

Comment:
This is not efficient

Reply:
Repeat, it has been judged at the entry of the code block

Reply:
> This is not efficient



> This is not efficient

Repeat, it has been judged at the entry of the code block

Reply:
noticed